TeachingRecordSystem/src/TeachingRecordSystem.SupportUi/Pages/Alerts/Alert/Index.cshtml

```diff
@@ -1,7 +1,65 @@
 @page "/alerts/{alertId}"
 @model IndexModel
 @{
-    ViewBag.Title = "Alert";
+    ViewBag.Title = Model.Alert!.Description;
 }
 
-<h1 class="govuk-heading-l">@ViewBag.Title</h1>
+@section BeforeContent {
+    <govuk-back-link href="@LinkGenerator.PersonAlerts(Model.PersonId!.Value)">Back</govuk-back-link>
+}
+
+<h1 class="govuk-heading-l" data-testid="title">@ViewBag.Title</h1>
+
+<div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds-from-desktop">
+        <table class="govuk-table trs-table--no-border govuk-!-margin-bottom-1">
+            <thead class="govuk-table__head">
+                <tr class="govuk-table__row">
+                    <th scope="col" class="govuk-table__header trs-table__header--no-border">Start date</th>
+                    <th scope="col" class="govuk-table__header trs-table__header--no-border">End date</th>
+                    <th scope="col" class="govuk-table__header trs-table__header--no-border">Status</th>
+                </tr>
+            </thead>
+            <tbody class="govuk-table__body">
+                <tr class="govuk-table__row" data-testid="alert-header">
+                    <td class="govuk-table__cell trs-table__cell--no-border" data-testid="start-date" use-empty-fallback>@(Model.Alert.StartDate.HasValue ? Model.Alert.StartDate.Value.ToString("dd/MM/yyyy") : string.Empty)</td>
+                    <td class="govuk-table__cell trs-table__cell--no-border" data-testid="end-date" use-empty-fallback>@(Model.Alert.EndDate.HasValue ? Model.Alert.EndDate.Value.ToString("dd/MM/yyyy") : string.Empty)</td>
+                    <td class="govuk-table__cell trs-table__cell--no-border" data-testid="status">@Model.Alert.Status</td>
+                </tr>
+            </tbody>
+        </table>
+    </div>
+</div>
+
+<hr class="govuk-section-break govuk-section-break--l govuk-section-break--visible">
+
+<div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds-from-desktop">
+        <h3 class="govuk-heading-s">Details</h3>
+        @if (!string.IsNullOrEmpty(@Model.Alert.Details))
+        {
+            <p class="govuk-body" data-testid="alert-details">
+            @{var lines = @Model.Alert.Details.Split(new string[] { "\r\n", "\r", "\n" }, StringSplitOptions.None);
```

Comment:
Can we move this into a tag helper to remove the duplication with `/persons/{id}/alerts`?

Reply:
Done